### PR TITLE
Refactor/planet interactives

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?
Adds more interactive features on planet info

- [ ] Fix
- [x] Feature
- [x] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Adds items for length of day, year, and diameter for users to interact with

### What ticket(s) do the changes resolve?
N/A
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/A

### How to Test Changes?
Tests were not written yet in Cypress

### CODE REVIEW

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 1]
```diff
@@ -14,26 +14,56 @@ import {
...
+
   const handleInput = (e: React.KeyboardEvent<HTMLInputElement>): void => {
     if (e.key === "e" || e.key === "." || e.key === "+" || e.key === "-" || e.key === "E") {
       e.preventDefault();
     }
   }
 
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-    if (Number(event.target.value) > 9999999) {
+  const formatWord = (word: string, input: number): string => {
```
Comment (alexthompson207) [inline src/components/PlanetInfo/PlanetInfo.tsx]: this is great

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 2]
```diff
@@ -147,7 +183,69 @@ const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet, resetSort }): JSX.Elem
...
+                  {diamInput && formatLargeNumbers(diamConversion(diameter))}{!diamInput && diamConversion(diamInput)} miles
+                </span>
+                {' '} away on {name}!
+              </p>
+            }
+          </div>
+
+          <div className='planet-info-card planet-info-card__moons'>
+            <img className='info-icon info-icon__moons' alt='eclipse icon' src='/space/eclipse-fill.svg'></img>
+            {formatMoons(name, number_of_moons)}
```
Comment (alexthompson207) [inline src/components/PlanetInfo/PlanetInfo.tsx:245]: I really like the new interactive features, good work!

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 3]
```diff
@@ -14,26 +14,56 @@ import {
...
       e.preventDefault();
     }
   }
 
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-    if (Number(event.target.value) > 9999999) {
+  const formatWord = (word: string, input: number): string => {
+    const finalWord: string = input === 1 ? word : `${word}s`;
+    return finalWord;
+  }
```
Comment (knees4bees) [inline src/components/PlanetInfo/PlanetInfo.tsx]: Right on. I'll make an issue for this so we remember to add this polish later.

**Review by alexthompson207 — APPROVED** [review]
We may need to delete some of the unused helper functions now?

**Review by knees4bees — APPROVED** [review]
Nice work, team!

**Pull request merged**